File: java/src/questao_7/Main.java
package questao_7;

public class Main {
    public static void main(String[] args) {
        System.out.println("Numeros entre 1000 e 2000 que quando divididos por 5 produzam resto 3");
        
        for (int n = 1000; n < 2000; n++) {
            if (n % 5 == 3) {
                System.out.println(n);
            }
        }
    }
}

File: python/src/questao_7/main.py
def main():
   print("Numeros entre 1000 e 2000 que quando divididos por 5 produzam resto 3")
   
   for n in range(1000,2000):
       if n % 5 == 3:
        print(n)
 

if __name__ == '__main__':
    main()